File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/prototype/transport/Car.java
package com.bigboxcode.designpattern.prototype.transport;

public class Car implements Prototype {
    public int make;
    public String model;
    public String color;

    public Car() {

    }

    public Car(Car car) {
        this.make = car.make;
        this.model = car.model;
        this.color = car.color;
    }

    @Override
    public Prototype clone() {
        return new Car(this);
    }

    public String toString() {
        return "Make: " + make + " | Model: " + model + " | Color: " + color;
    }

}

File: design-patterns/php/abstractfactory/transport/Car.php
<?php
// Car.php

namespace BigBoxCode\DesignPattern\AbstractFactory\Transport;

class Car implements Transport {
    public function start(): void {
        echo "Car Started\n";
    }

    public function stop(): void {
        echo "Car Stopped\n";
    }

    public function repair(): void {
        echo "Car Repair\n";
    }
}